locales/ar.yml

```diff
@@ -0,0 +1,11 @@
+---
+ar:
+  header:
+    title: "مدوّنة برلين للسلوك"
+    sub-title: "مدوّنة سلوك لكل المجموعات و الفعاليات"
+  supporters:
+    title: "المتبنّون"
+    sub-title: "المجموعات و الفعاليات التالية تتبنّى مدونة برلين للسلوك:"
```

Comment:
I guess it would be better to lead with the verb here, Like:

```
تتبنّى المجموعات و الفعاليات التالية مدونة برلين للسلوك 
```

Reply:
Yes! I forgot about the RTL issue. Please do what it takes to make it work. Thanks, Amr.

Reply:
Regarding leading with the verb, there's a common perception that it is superior to starting a sentence with a noun. It is not. In Arabic a noun phrase and a verb phrase are both correct. You might remember having studied both in school :)
The governing factor for me here is clarity and disambiguity. With most modern Arabs not depending on declination to decode the relationships between nouns, consecutive nouns  become harder to decode and writers resort to inserting unnecessary connecting articles to separate nouns and clarigy the relationships between them. This phenomena is happening to several languages, not only Arabic (German for example). It leads to awkward and less eloquent constructs. All of this can be avoided by altering the word order, and in this case it entails starting with a noun in order to have the verb as a natural separator between nouns.